.circleci/config.yml

```diff
@@ -279,7 +279,7 @@ jobs:
           name: Install pipenv
           command: pip install --upgrade pip pipenv
       - restore_cache:
-          key: flowdb-deps-1-{{ checksum "flowdb/tests/Pipfile.lock"}}
+          key: flowdb-deps-6-{{ checksum "flowdb/tests/Pipfile.lock"}}
```

Comment:
Do we want this to be in lockstep with `flowmachine-deps`, on so we can bump them all at once if we need to with something like `s/deps-6-/deps-7-/g`?

Reply:
Could do - but then we'd end up bumping all the caches any time one of them needs bumping, which seems excessive.